build-tools/automation/azure-pipelines.yaml

```diff
@@ -759,7 +759,7 @@ stages:
       inputs:
         github_token: $(GitHub.Token)
         provisioning_script: $(System.DefaultWorkingDirectory)/designer/bot-provisioning/dependencies.csx
-        provisioning_extra_args: -remove Xamarin.Android -vv
+        provisioning_extra_args: -remove Xamarin.Android -vv -f
```

Comment:
I don't want to force install all of the designer dependencies, as their profiles contain things we likely don't want to wait to re-install on every run. As long as the default parameters in `run-installer.yaml` are updated to force re-installation of `Xamarin.Android`, then we don't need to modify the designer provisionator logic.